be/src/models/dbo.models.ts

```diff
@@ -0,0 +1,22 @@
+import {ObjectId} from "mongodb";
```

Comment:
Should the filename not be `dto.models.ts`?

Reply:
No, as mentioned in https://github.com/TailoredMediaProject/tm-knowledge-store/pull/9#discussion_r765921081, these are `DBO`s.